pages/index.js

```diff
@@ -1,144 +1,273 @@
-          </div>
-        </Col>
-        <Col xl="1" className="d-none d-xl-block" />
-        <Col md="6" xl="5" className="d-none d-lg-block">
-          <RandomQueue />
-        </Col>
-      </Row>
-    </Container>
-    <Container className="mt-4">
-      <Row className="justify-content-center">
-        <Col xl="6" lg="8">
-          <video className="dummy-video mx-auto mb-4 d-block" controls>
-            <source src="https://queue.illinois.edu/hosted/queue-video.mp4" type="video/mp4" />
-            <source src="https://queue.illinois.edu/hosted/queue-video.webm" type="video/webm" />
-            https://queue.illinois.edu/hosted/queue-video.mp4
-          </video>        
-        </Col>
-      </Row>
-      <Row>
-        <Col md="6" className="mb-3">
-          <h4><FontAwesomeIcon icon={faUsers} size="lg" className="mr-2" fixedWidth />Open Office Hours</h4>
-          <p className="mb-4">
-            The Queue eliminates the need for hand-raising and trying to keep track of who raised their hand first. If an instructor has overflowing office hours,
-            instructors can allow students can let the instructor know where they are within a nearby area, and what they need help with.
-          </p>
-        </Col>
+export default () => {
+  const blogPost = (blogItemsMeta || []).find(post => {
+    return post.post.meta.featured
+  })
 
-        <Col md="6" className="mb-3">
-          <h4><FontAwesomeIcon icon={faChalkboardTeacher} size="lg" className="mr-2" fixedWidth />Group Formation and Active Learning</h4>
-          <p className="mb-4">
-            Instructors can use the Queue for group formation and explanation. By grouping students together based on the topic they are having trouble with,
-            the instructor doesn&apos;t have to explain the same concept to several students separately. While in class, students can post questions in the queue
-            so that instructors can better prepare themselves prior to helping them.
-          </p>
-        </Col>
-        
-        <Col md="6" className="mb-3">
-          <h4><FontAwesomeIcon icon={faBolt} size="lg" className="mr-2" fixedWidth />Real Time</h4>
-          <p className="mb-4">
-            Students can monitor their position in a queue without needing to refresh the page. And instructors
-            will get notifications whenever someone new joins the queue so they can help students as quickly as possible.
-          </p>
-        </Col>
+  return (
+    <>
+      <Container fluid className="bg-dark py-3">
+        <Row className="justify-content-center">
+          <Col
+            sm="10"
+            md="8"
+            lg="6"
+            className="d-flex flex-column justify-content-center py-3 py-md-5 py-lg-0 text-center"
+          >
+            <h1 className="text-light mb-4">Queue@Illinois</h1>
+            <h4 className="text-light mb-4">
```

Comment:
I think this is just my personal preference, but I like it better if it's h5 rather than h4.  I also like a padding of 5 on the entire container.

![image](https://user-images.githubusercontent.com/32113753/54326500-851a7e00-45d4-11e9-8b01-9604160198c6.png)